src/app/contact/contact.component.html

```diff
@@ -0,0 +1,339 @@
+              <option data-countryCode="SM" value="378">San Marino (+378)</option>
+              <option data-countryCode="ST" value="239">Sao Tome &amp; Principe (+239)</option>
+              <option data-countryCode="SA" value="966">Saudi Arabia (+966)</option>
+              <option data-countryCode="SN" value="221">Senegal (+221)</option>
+              <option data-countryCode="CS" value="381">Serbia (+381)</option>
+              <option data-countryCode="SC" value="248">Seychelles (+248)</option>
+              <option data-countryCode="SL" value="232">Sierra Leone (+232)</option>
+              <option data-countryCode="SG" value="65">Singapore (+65)</option>
+              <option data-countryCode="SK" value="421">Slovak Republic (+421)</option>
+              <option data-countryCode="SI" value="386">Slovenia (+386)</option>
+              <option data-countryCode="SB" value="677">Solomon Islands (+677)</option>
+              <option data-countryCode="SO" value="252">Somalia (+252)</option>
+              <option data-countryCode="ZA" value="27">South Africa (+27)</option>
+              <option data-countryCode="ES" value="34">Spain (+34)</option>
+              <option data-countryCode="LK" value="94">Sri Lanka (+94)</option>
+              <option data-countryCode="SH" value="290">St. Helena (+290)</option>
+              <option data-countryCode="KN" value="1869">St. Kitts (+1869)</option>
+              <option data-countryCode="SC" value="1758">St. Lucia (+1758)</option>
+              <option data-countryCode="SD" value="249">Sudan (+249)</option>
+              <option data-countryCode="SR" value="597">Suriname (+597)</option>
+              <option data-countryCode="SZ" value="268">Swaziland (+268)</option>
+              <option data-countryCode="SE" value="46">Sweden (+46)</option>
+              <option data-countryCode="CH" value="41">Switzerland (+41)</option>
+              <option data-countryCode="SI" value="963">Syria (+963)</option>
+              <option data-countryCode="TW" value="886">Taiwan (+886)</option>
+              <option data-countryCode="TJ" value="7">Tajikstan (+7)</option>
+              <option data-countryCode="TH" value="66">Thailand (+66)</option>
+              <option data-countryCode="TG" value="228">Togo (+228)</option>
+              <option data-countryCode="TO" value="676">Tonga (+676)</option>
+              <option data-countryCode="TT" value="1868">Trinidad &amp; Tobago (+1868)</option>
+              <option data-countryCode="TN" value="216">Tunisia (+216)</option>
+              <option data-countryCode="TR" value="90">Turkey (+90)</option>
+              <option data-countryCode="TM" value="7">Turkmenistan (+7)</option>
+              <option data-countryCode="TM" value="993">Turkmenistan (+993)</option>
+              <option data-countryCode="TC" value="1649">Turks &amp; Caicos Islands (+1649)</option>
```

Comment:
`&amp` ? jQuery? jQuery is not defined here. That is why maybe you're getting Travis error.